subsys/bluetooth/controller/ll_sw/nordic/lll/lll_sync_iso.c

```diff
@@ -0,0 +1,728 @@
+	uint32_t hcto;
+	uint8_t phy;
+
+	DEBUG_RADIO_START_O(1);
+
+	lll = p->param;
+
+	/* Deduce the latency */
+	lll->latency_event = lll->latency_prepare - 1;
+
+	/* Calculate the current event counter value */
+	event_counter = (lll->payload_count / lll->bn) +
+			(lll->latency_event * lll->bn);
+
+	/* Update BIS packet counter to next value */
+	lll->payload_count += (lll->latency_prepare * lll->bn);
+
+	/* Reset accumulated latencies */
+	lll->latency_prepare = 0;
+
+	/* Current window widening */
+	lll->window_widening_event_us += lll->window_widening_prepare_us;
+	lll->window_widening_prepare_us = 0;
+	if (lll->window_widening_event_us > lll->window_widening_max_us) {
+		lll->window_widening_event_us =	lll->window_widening_max_us;
+	}
+
+	/* Initialize to mandatory parameter values */
+	lll->bis_curr = 1U;
+	lll->ptc_curr = 0U;
+	lll->irc_curr = 1U;
+	lll->bn_curr = 1U;
+
+	/* Initialize control subevent flag */
+	lll->ctrl = 0U;
+
+	/* Initialize trx chain count */
+	trx_cnt = 0U;
+
+	/* Calculate the Access Address for the BIS event */
+	util_bis_aa_le32(lll->bis_curr, lll->seed_access_addr, access_addr);
+	data_chan_id = lll_chan_id(access_addr);
+
+	/* Calculate the CRC init value for the BIS event,
+	 * preset with the BaseCRCInit value from the BIGInfo data the most
+	 * significant 2 octets and the BIS_Number for the specific BIS in the
+	 * least significant octet.
+	 */
+	crc_init[0] = lll->bis_curr;
+	(void)memcpy(&crc_init[1], lll->base_crc_init, sizeof(uint16_t));
+
+	/* Calculate the radio channel to use for ISO event */
+	data_chan_use = lll_chan_iso_event(event_counter, data_chan_id,
+					   &lll->data_chan_map[0],
+					   lll->data_chan_count,
+					   &lll->data_chan_prn_s,
+					   &lll->data_chan_remap_idx);
+	lll->ctrl_chan_use = data_chan_use;
+
+	/* Start setting up of Radio h/w */
+	radio_reset();
+
+#if defined(CONFIG_BT_CTLR_TX_PWR_DYNAMIC_CONTROL)
+	radio_tx_power_set(lll->tx_pwr_lvl);
+#else
+	radio_tx_power_set(RADIO_TXP_DEFAULT);
+#endif
+
+	phy = lll->phy;
+	radio_phy_set(phy, 1);
+	radio_pkt_configure(8, lll->max_pdu, (phy << 1));
+	radio_aa_set(access_addr);
+	radio_crc_configure(PDU_CRC_POLYNOMIAL, sys_get_le24(crc_init));
+	lll_chan_set(data_chan_use);
+
+	node_rx = ull_pdu_rx_alloc_peek(1);
+	LL_ASSERT(node_rx);
+	radio_pkt_rx_set(node_rx->pdu);
+
+	radio_switch_complete_and_disable();
+
+	ticks_at_event = p->ticks_at_expire;
+	ull = HDR_LLL2ULL(lll);
+	ticks_at_event += lll_event_offset_get(ull);
+
+	ticks_at_start = ticks_at_event;
+	ticks_at_start += HAL_TICKER_US_TO_TICKS(EVENT_OVERHEAD_START_US);
+
+	remainder = p->remainder;
+	remainder_us = radio_tmr_start(0, ticks_at_start, remainder);
+
+	radio_tmr_ready_save(remainder_us);
+	radio_tmr_aa_save(0);
+	radio_tmr_aa_capture();
+
+	hcto = remainder_us +
+	       ((EVENT_JITTER_US + EVENT_TICKER_RES_MARGIN_US +
+		 lll->window_widening_event_us) << 1) +
+	       lll->window_size_event_us;
+	hcto += radio_rx_ready_delay_get(lll->phy, 1);
```

Comment:
Not sure if I commented on this in another PR, but the second parameter here should probably also use a `#define`d value.